writing/kill-the-dog.md

```diff
@@ -0,0 +1,25 @@
+# Killing the Dog - My new reason for practicing mindfulness
+
+Anyone else seriously wanted to kill their dog ☠️🐶? And gone through a bunch of plans to make it seem like an accident?
+
+- "Accidentally" leaving the front door open so she runs into the road
+- Host a house party 🥳, put on "Who let the dogs out", turn it up and keep putting the puppy in the middle of the dance floor until someone crushes her to death with their funky moves
+- Travel back to the Roman times, tell everyone the dog is the Messiah and wait until the inevitable crucifixion ✟
```

Comment:
Stay consistent with verb tense on the three bullets... the first bullet works fine, then you need to adjust the other two... "hosting... putting on.... turning it up" etc.